app/views/items/browse_facet.html.erb

```diff
@@ -20,7 +19,8 @@
     <% end %>
   </ul>
 </nav>
-
+<p>Select facets listed below, then click "search" to see search results</p>
+<%= link_to "Search", prefix_path("home_path", f: params[:f]), rel: "nofollow", class: "btn btn-default" %>
```

Comment:
Again, an explanation in the commit message would help. I think we talked about this in a dev meeting and y'all said it was done so people don't have to scroll all the way to the bottom of a list with thousands of facets, but it should be documented with the code change so it doesn't appear arbitrary to a code reviewer - both when reviewing the PR for merge now and later if we use git history to look back on when / why the code was changed.